_pages/dev/performance/deferred-loading.md

```diff
@@ -56,16 +56,18 @@ ConfigModule.withConfig(<CmsConfig>{
 });
 ```
 
-Components can also be configured to use `DEFER` loading strategy while the global strategy is set to `INSTANT`.
+**Note:** Components can also be configured to use the `DEFER` loading strategy while the global strategy is set to `INSTANT`.
 
-### SSR
+### Server-Side Rendering
 
-Deferred loading does not applied to SSR, so that crawlers will receive the full DOM.
+Deferred loading is not applied to server-side rendering, so that crawlers can receive the full DOM.
 
 ## Related Techniques
 
-There are a number of related techniques that we like to highlight:
+There are a number of related techniques to be aware of:
 
--   Lazy loading of JS chunks – Deferred Loading does _not_ lazy-load the component JS bundles; it is only concerned with the creation / rendering of the components.
--   Above-the-fold loading – Deferred Loading is tightly coupled to so-called [Above-the-Fold Loading]({{ site.baseurl }}{% link _pages/dev/performance/above-the-fold.md %}), but this is documented in another section.
--   Deferred Loading is not related to the native lazy loading of images which has recently landed in Chrome. This new API is not yet fully supported, but could be used in combination with deferred loading for CMS components.
+- Lazy loading of JS chunks: Deferred loading does not lazy-load the component JS bundles. Deferred loading is only concerned with the creation or rendering of components.
+
+- Above-the-fold loading: Deferred loading is tightly coupled to so-called "above-the-fold loading". For more information, see [Above-the-Fold Loading]({{ site.baseurl }}{% link _pages/dev/performance/above-the-fold.md %}).
+
+- Native lazy loading of images: Deferred loading is not related to the native lazy loading of images that has recently been implemented in Chrome. This new API is not yet fully supported, but could be used in combination with deferred loading for CMS components.
```

Comment:
can we change this to 
" that has recently been implemented in some browsers. "

Reply:
Done.